python/ray/tests/test_tempfile.py

```diff
@@ -42,7 +42,7 @@ def test_tempdir_commandline():
     shutil.rmtree(ray.utils.get_ray_temp_dir(), ignore_errors=True)
     check_call_ray([
         "start", "--head", "--temp-dir=" + os.path.join(
-            ray.utils.get_user_temp_dir(), "i_am_a_temp_dir2")
+            ray.utils.get_user_temp_dir(), "i_am_a_temp_dir2"), "--port", "0"
```

Comment:
Using port 0 feels a little scary here (it's reserved and sometime requires root permissions to bind to), can we pick something >= 1024?

Reply:
port 0 normally means give me whatever is available in the system, like when you create a socket and pass 0, it gives you an available port.

Reply:
it is a convention, there is no port 0 existing.